Update velocity shutdown command (PaperMc/Velocity#1427) (#493)
PaperMC/docs: docs/velocity/admin/reference/commands.md

## `/shutdown`

:::info

This command can only be executed from console.

:::

When run, this will gracefully shut down the Velocity proxy.
All players will be disconnected from the proxy, and plugins will have a chance to finish up before the proxy shuts down.
An optional reason can be given, either as JSON or with [MiniMessage Format](https://docs.advntr.dev/minimessage/format.html).

> If the message provided starts with `"`, `[`, or `{`, Then the message will tried to be parsed as JSON.

Reviewer: ```suggestion
If the provided message starts with `"`, `[`, or `{`, then the message is attempted to be parsed as JSON.
```

If this parsing fails, or the message starts with anything else, the message will be parsed as MiniMessage.

> 

Reviewer 2: Extra newline
